FE/airbnb-fe/src/components/utils/reducer/modalClickReducer.js

```diff
@@ -0,0 +1,33 @@
+const modalClickReducer = (state, action) => {
+  switch (action.type) {
+    case 'CHECKINOUT':
+      return {
+        checkInOut: !state.checkInOut,
+        price: false,
+        people: false,
+      };
+    case 'PRICE':
+      return {
+        checkInOut: false,
+        price: !state.price,
+        people: false,
+      };
+    case 'PEOPLE':
+      return {
+        checkInOut: false,
+        price: false,
+        people: !state.people,
+      };
+    case 'ModalOff':
```

Comment:
이 친구도 다른 action.type들처럼 MODAL_OFF여도 괜찮을 것 같습니다.

```suggestion
    case 'MODAL_OFF':
```